### PR TITLE
Staging/1.0.0 v2

### PR DESCRIPTION
**Pull request by phalseth-hpe**
This PR represents documentation changes to properly change existing references to "1.0" to "1.0.1", and existing references to "1.0.1" to "1.0.10".

### CODE REVIEW

**Review by rustydb — APPROVED** [review]

**Review by jasonrouault — COMMENTED** [review]
see inline comments

File: upgrade/index.md [hunk 1]
```diff
@@ -27,22 +27,22 @@ See [Prepare for Upgrade](prepare_for_upgrade.md)
...
-* Upgrading **to CSM 1.0.0**
+* Upgrading **to CSM 1.0.1**
 
     The upgrade of CSM software will do a controlled, rolling reboot of all management nodes before updating the CSM services.
 
-    The upgrade is a guided process starting with [Upgrade Management Nodes and CSM Services](1.0/README.md).
+    The upgrade is a guided process starting with [Upgrade Management Nodes and CSM Services](1.0.1/README.md).
 
-* Upgrading **to CSM 1.0.1**
+* Upgrading **to CSM 1.0.2**
```
Comment (jasonrouault) [inline upgrade/index.md:36]: should be 1.0.10 vs 1.0.2

File: upgrade/index.md [hunk 2]
```diff
@@ -27,22 +27,22 @@ See [Prepare for Upgrade](prepare_for_upgrade.md)
...
    
-    **IMPORTANT**: You must already be at CSM 1.0.0 in order to upgrade to CSM 1.0.1.
+    **IMPORTANT**: You must already be at CSM 1.0.1 in order to upgrade to CSM 1.0.2.
```
Comment (jasonrouault) [inline upgrade/index.md:38]: should be 1.0.10 vs 1.0.2

File: upgrade/index.md [hunk 3]
```diff
@@ -27,22 +27,22 @@ See [Prepare for Upgrade](prepare_for_upgrade.md)
...
   
-    The upgrade is a guided process starting with [CSM 1.0.1 Patch Installation Instructions](1.0.1/README.md).
+    The upgrade is a guided process starting with [CSM 1.0.2 Patch Installation Instructions](1.0.2/README.md).
```
Comment (jasonrouault) [inline upgrade/index.md:40]: should be 1.0.10 vs 1.0.2

File: upgrade/index.md (hunk 2 above)
Comment (phalseth-hpe) [reply upgrade/index.md]: fixed, thanks

File: upgrade/index.md (hunk 3 above)
Comment (phalseth-hpe) [reply upgrade/index.md]: fixed, thanks

**Comment by bklei** [comment]
Found a few other instances of `csm-1.0.0` in the branch:

```
C02YT3JRLVCJ:docs-csm bklein$ find . -type f | xargs grep csm-1.0 | grep -v 1.0.1
./background/ncn_boot_workflow.md:> **`NOTE`** Cloud-init will set boot order when it runs, but this does not always work with certain hardware vendors. An administrator can invoke the cloud-init script at `/srv/cray/scripts/metal/set-efi-bbs.sh` on any NCN. Find the script [here, on GitHub](https://github.com/Cray-HPE/node-image-build/blob/lts/csm-1.0/boxes/ncn-common/files/scripts/metal/set-efi-bbs.sh).
./operations/utility_storage/Restore_Corrupt_Nexus.md:   Locate the Nexus setup script, this is typically in `/root/csm-1.0.*` on `ncn-m001`:
./operations/utility_storage/Restore_Corrupt_Nexus.md:   ncn-m001# ls /root/csm-1.0.0-beta.50/lib/setup-nexus.sh
./operations/utility_storage/Restore_Corrupt_Nexus.md:   /root/csm-1.0.0-beta.50/lib/setup-nexus.sh
./operations/utility_storage/Restore_Corrupt_Nexus.md:   ncn-m001# /root/csm-1.0.0-beta.50/lib/setup-nexus.sh
```

```
C02YT3JRLVCJ:docs-csm bklein$ git status
On branch staging/1.0.0-v2
Your branch is up to date with 'origin/staging/1.0.0-v2'.
```

**Comment by phalseth-hpe** [comment]
@bklei I'm not sure the ncn_boot_workflow.md instance needs a change, because it is referencing the "csm-1.0" branch in an the node_image_build repository, but I will update the instances in Restore_Corrupt_Nexus.md.

**Review by bklei — APPROVED** [review]

**Pull request merged**